src/main/java/wooteco/subway/domain/Distance.java

```diff
@@ -0,0 +1,58 @@
+package wooteco.subway.domain;
+
+public class Distance {
+    private final int value;
+
+    public Distance(int value) {
+        this.value = value;
+        validatePositive(value);
+    }
```

Comment:
개인적으로 값이 할당되기 전에 검증하는걸 선호하는 편입니다!
원자성을 갖을 수 있을 것 같아요.

```suggestion
    public Distance(int value) {
        validatePositive(value);
        this.value = value;
    }
```

Reply:
동시성 이슈를 해결하기 위해 코드를 위와 같이 작성했는데, 지금 생각해보니 참조값이 아니라 원시값이라서 검증과 할당 사이에 값이 바뀔 걱정을 하지 않아도 괜찮군요 😅